README.md

```diff
@@ -16,13 +16,13 @@ Then start Rocket.Chat linked to this mongo instance:
 
     docker run --name rocketchat --link db -d rocket.chat
 
-This will start a Rocket.Chat instance listening on the default Meteor port of 3000 on the container.
+This will start a Rocket.Chat instance listening on the default Meteor port of 3000 on the container.  You can then link a web server to front-end Rocket.Chat.
```

Comment:
But I think only 'hosters' (minority) can use this first sample config.

Most users trying out Rocket.Chat just want to get it running.  And there is a missing implied step to 'get at' the already running server.